### PR TITLE
Demo with only one table

### PR DESCRIPTION
**Pull request by laurgarn**

### CODE REVIEW

File: demo/src/TableTab.js [hunk 1]
```diff
@@ -128,6 +128,12 @@ export const TableTab = () => {
         return ret;
     }, []);
 
+    const [isIndexerExternal, setIndexerIsExternal] = useState(true);
+    const [sortable, setSortable] = useState(true);
+    const [recreats, setRecreats] = useState(false);
```
Comment (jonenst) [inline demo/src/TableTab.js:133]: recreates

File: demo/src/TableTab.js [hunk 2]
```diff
@@ -161,82 +167,114 @@ export const TableTab = () => {
...
+        return (
+            <Stack sx={{ margin: '1ex' }}>
+                <FormControlLabel
+                    control={
+                        <Switch
+                            checked={usesCustomStyles}
+                            onChange={() => setUsesCustomStyles((was) => !was)}
+                        />
+                    }
+                    // labelPlacement={'start'}
```
Comment (jonenst) [inline demo/src/TableTab.js:185]: remove instead of comment

File: demo/src/TableTab.js (hunk 1 above)
Comment (laurgarn) [reply demo/src/TableTab.js]: done

File: demo/src/TableTab.js (hunk 2 above)
Comment (laurgarn) [reply demo/src/TableTab.js]: done

**Review by jonenst — APPROVED** [review]

**Pull request merged**